web-admin/src/routes/[organization]/-/settings/+layout.svelte

```diff
@@ -19,12 +19,14 @@
   $: hideUsageSettings = onEnterprisePlan || !billingPortalUrl;
 
   $: navItems = [
-    { label: "General", route: "" },
+    { label: "General", route: "", hasPermission: true },
```

Comment:
What about `hasPermission` could default to `true` if it's not explicitly set?

Reply:
fyi: this aligns with the existing pattern in the codebase using `hasPermission` for the tabs https://github.com/rilldata/rill/blob/main/web-admin/src/features/projects/ProjectTabs.svelte#L17-L45

by default, if `hasPermission` isn't provided, it defaults to true.